Makefile

```
@@ -14,10 +14,10 @@ fmt:
 
 lint:
 ifndef HAS_LINT
-	go get -u golang.org/x/lint/golint
-	echo "installing golint"
+	go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
```

Should we point to the specific version here ?
```suggestion
	go get -u github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1
```

---

It makes sense!